applications/fax/src/fax_smtp.erl

```diff
@@ -516,7 +516,8 @@ check_empty_permissions(#state{errors=Errors
 
 -spec match(binary(), binary()) -> boolean().
 match(Address, Element) ->
-    re:run(Address, Element) =/= 'nomatch'.
+    Address =:= kz_term:to_lower_binary(Element)
```

Comment:
@swysor any thought to normalize Address too?

kz_term:to_lower_binary(Address) =:= kz_term:to_lower_binary(Element)

Reply:
its normalized on ingress

Reply:
https://github.com/2600hz/kazoo/blob/a77772f3513097e649fc3953c983915fe03efe71/applications/fax/src/fax_smtp.erl#L111